test/shared/utils.py

```diff
@@ -81,60 +81,61 @@ def expected_wheels(package_name, package_version, manylinux_versions=None,
     # {python tag} and {abi tag} are closely related to the python interpreter used to build the wheel
     # so we'll merge them below as python_abi_tag
 
+    if manylinux_versions is None:
+        if pm.machine() == 'x86_64':
+            manylinux_versions = ['manylinux1', 'manylinux2010']
+        else:
+            manylinux_versions = ['manylinux2014']
+
     python_abi_tags = ['cp35-cp35m', 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38']
-    extra_x86_python_abi_tags = ['cp27-cp27m', 'pp27-pypy_73', 'pp36-pypy36_pp73']
 
-    if platform == 'linux':
-        if pm.machine() not in ['x86_64', 'i686']:
-            if manylinux_versions is None:
-                manylinux_versions = ['manylinux2014']
-            architectures = {'cp': [pm.machine()]}
-        else:
-            if manylinux_versions is None:
-                manylinux_versions = ['manylinux1', 'manylinux2010']
-            python_abi_tags += extra_x86_python_abi_tags
+    if pm.machine() in ['x86_64', 'AMD64', 'x86']:
```

Comment:
Why not just the `['x86_64', 'i686']` from before?

Reply:
Ah, wait, I think I see. Because Windows and macOS give different values? :-|
So `'i686'` was never used, then?

Reply:
Correct, yes. `i686` was never used (you can't use Docker on i686). Windows refers to `x86_64` as `AMD64`. But Windows reports architecture weirdly - even on a 64-bit machine, if Python is running in 32-bit mode, it reports `x86`. I'm not sure if we have to support 32-bit Python, but I think 32-bit might be more common on Windows so I added that too.

Reply:
Nice, thanks!
Then let's just get it merged! :-)